advanced-usage/index.md

```diff
@@ -368,46 +401,157 @@ The File Browser allows viewing, editing, downloading, and uploading BlueOS file
 {% end %}
 {{ easy_image(src="nmea-injector", width=600, class="pirate") }}
 {% pirate() %}
-- Setup requires a UDP socket for the NMEA device to connect to, and a MAVLink ID
+- Setup requires a UDP or TCP socket for the NMEA device to connect to, and a MAVLink ID
 for the component that will send the location data to the vehicle
 {% end %}
 {{ easy_image(src="nmea-example", width=400, class="pirate") }}
 
+### Ping Sonar Devices
+{{ service(service="Ping Service", port=9110, link="/services/ping", based=true) }}
+
+`New in 1.1`
+
+The Ping Sonar Devices page shows any detected 
+[sonars](@/hardware/additional/sonars/index.md) from the Ping family,
+including [ethernet-configured Ping360s](https://bluerobotics.com/learn/changing-communications-interface-on-the-ping360/#wiring-connections--ethernet-configuration)
+that are visible on the local network (e.g. via an 
+[Ethernet Switch](@/hardware/additional/ethernet-switch/index.md)).
+
+- Allows configuring Ping Sonar distance estimates to send as MAVLink
+[`DISTANCE_SENSOR`](https://mavlink.io/en/messages/common.html#DISTANCE_SENSOR)
+messages to the autopilot, for viewing in the Control Station Software and
+logging as part of the telemetry stream
+- Provides a viewing utility for devices connected via USB/serial, to show
+which port they are plugged into 
+
+{{ easy_image(src="ping-sonar-devices", width=600) }}
+
 {% pirate() %}
-### Terminal
+### Serial Bridges
 {% end %}
-{{ service(service="ttyd" link="https://tsl0922.github.io/ttyd/", port=8088) }}
+{{ service(service="Bridget", port=27353, link="/services/bridget", based=true) }}
 {% pirate() %}
 
-The terminal provides
-- A tmux session
-- Direct access into the core BlueOS docker container
-- Ready access to the tmux sessions of the core services (`CTRL+b s`)
-   - Useful for seeing logs as they update live
-   - Can kill services if necessary
-- Access to the underlying device via the `red-pill` utility
-   - Can return to the docker using the `exit` command, or pressing `CTRL+d`
-{% end %}
-{{ easy_image(src="terminal", width=600, class="pirate") }}
+The Serial Bridges page allows creating high performance links between serial
```

Comment:
we should make it clear that this cannot be used directly to connect things to the flight controller